extensions/wikia/AffiliateService/js/ext.AffiliateService.js

```diff
@@ -53,8 +53,22 @@ require([
 	var AffiliateService = {
 		$infoBox: undefined,
 
+		// ?debugAffiliateServiceTargeting=campaign,category
+		getDebugTargeting: function() {
+			// check if we have the mechanism to get the param (ie. not on IE)
+			if (typeof URLSearchParams === 'function') {
+				var urlParams = new URLSearchParams(w.location.search);
+
+				if (urlParams.has('debugAffiliateServiceTargeting')) {
```

Comment:
Any reason to use a different param than we are on mobile ?

Reply:
The service/extension name is different? And this is explicitly just faking the targeting. I'm not tied to that, I can change it if you like.

Reply:
Your call. I just think it would be easier to have a single type of query param.